<launch>
    <arg name="grasp_planner" default="gascan"/>
    <include file="$(find grasping_executive)/config/$(arg grasp_planner)_grasp_planner.launch.xml"/>
</launch>
